src/python/grpcio_tests/tests_aio/unit/_test_base.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import unittest
+from grpc.experimental import aio
+
+
+class AioTestBase(unittest.TestCase):
+
+    def setUp(self):
+        self._loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(self._loop)
+        aio.init_grpc_aio()
+
+    @property
```

Comment:
Is this necessary? Seems like you could just use `asyncio.get_running_loop()`.

Reply:
It is not necessary, I found it is easier to type `self.loop` than `asyncio.get_event_loop()`, and theoretically a bit faster. Also, `get_running_loop` is 3.7+.

One small special difference is that at 3.5.0 & 3.5.1 (which we don't support), the `get_event_loop` has bug (https://github.com/python/asyncio/pull/452).